src/shallot.py

```diff
@@ -9,6 +9,9 @@
 import crypto
 import list_server
 from config import SHALLOT
+from config import X25519_SIZE
```

Comment:
You probably want to call `CRYPTO` from config here. This doesn't import well.

Reply:
Resolved this